src/Data/Json/Extended.purs

```diff
@@ -0,0 +1,126 @@
+module Data.Json.Extended
+  ( module Sig
+
+  , EJson(..)
+  , getEJson
+
+  , class EJsonView
+  , intoEJson
+  , outEJson
+
+  , renderEJson
+
+  , arbitraryEJsonOfSize
+  , arbitraryJsonEncodableEJsonOfSize
+  ) where
+
+import Prelude
+
+import Data.Eq1 (eq1)
+import Data.Argonaut.Core as JS
+import Data.Argonaut.Encode (class EncodeJson)
+import Data.Argonaut.Decode (class DecodeJson)
+
+import Data.Either as E
+import Data.Functor.Mu as Mu
+import Data.Maybe as M
+
+import Test.StrongCheck as SC
+import Test.StrongCheck.Gen as Gen
+
+import Data.Json.Extended.Signature as Sig
+
+class EJsonView a where
```

Comment:
At first, it seemed like this would be very useful, but I'm not really certain it's worth including here. I'll probably get rid of it.